ir/src/tests/mod.rs

```diff
@@ -379,6 +379,126 @@ fn transition_constraints_using_parens() {
     assert!(result.is_ok());
 }
 
+#[test]
+fn random_values_fixed_list() {
```

Comment:
nit: Should this be renamed to `random_values_indexed_access` or something like that?